src/ts/contentScripts/live-preview/index.tsx

```diff
@@ -0,0 +1,173 @@
+import {Feature, Settings} from '../../utils/settings'
+import {RoamDate} from '../../date/common'
+import {Navigation} from '../../roam/navigation'
+import {browser} from 'webextension-polyfill-ts'
+import {createPopper, Instance} from '@popperjs/core'
+
+export const config: Feature = {
+    id: 'live_preview',
+    name: 'Live Preview',
+    settings: [],
+}
+
+Settings.isActive('live_preview').then(active => {
+    if (active) {
+        enableLivePreview()
+    }
+})
+
+browser.runtime.onMessage.addListener(async message => {
+    if (message?.featureId === 'live_preview' && message?.type === 'toggle') {
+        if (message.value) {
+            enableLivePreview()
+        } else {
+            removeLivePreview()
+        }
+    }
+})
+
+const createPreviewIframe = () => {
+    const iframe = document.createElement('iframe')
+    const currentDate = new Date()
+    // Tomorrow starts at 2am for the purposes of this
+    currentDate.setHours(currentDate.getHours() - 2)
```

Comment:
why do you need this here?

Reply:
this is for the first page that we load by default -- I had kept it as 'today', but in the userscript I made, I kept this the 'search' page. I will do that here too

Reply:
I think you can just use "baseUrl"

Reply:
@Stvad just the base url is /page, which is not a valid page. Refactored to using daily notes

Reply:
not `basePageUrl` but `baseUrl` which would be something like `https://roamresearch.com/#/app/roam-toolkit` and will lend you on the daily notes